docs/release_process.md

```diff
@@ -0,0 +1,40 @@
+# Release process
+
+To release new versions of one or more packages in the workspace:
+
+1. Create a local branch and make these changes:
+   1. Update the version of each package you want to release in the
+      corresponding `Cargo.toml`. Follow semver.
+   2. If needed, update the dependency version in packages that depend
```

Comment:
Mabye konce one of these exists it might be nice to backfill a link to the example PR for this process.